tabbycat/utils/templatetags/debate_tags.py

```diff
@@ -183,6 +183,12 @@ def roundurl(parser, token):
     return RoundURLNode(*args)
 
 
+@register.simple_tag
+def person_display_name(person, tournament):
+    # If returned directly from the object it will have to lookup tournament
+    return person.get_public_name(tournament)
```

Comment:
Similarly - is it possible to get the `tournament` from `context`, in the same way that the `tournament_url` template tag does? (I'm not sure what the answer is... I mean, I'm sure there's a way, but I'm not sure how painful it'd be.) I notice that other analogous tags don't do this, again not sure if there's a reason for it or if we just didn't realise it was an idea at the time.)

Reply:
Yep, with the `takes_context` decorator argument!